src/datafactory/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.11.0
++++++
+* Support --selfcontainediaenabled in integration-runtime self-hosted create
```

Comment:
Pleas split words in argument option name by '-'

Reply:
fixed